templates/desktop/features.html

```diff
@@ -56,70 +56,88 @@ <h2>A whole world of apps</h2>
           <div class="p-matrix__content">
-            <h3 class="p-matrix__title">Telegram</h3>
-            <p class="p-matrix__desc">Mobile and desktop messaging app with a focus on security and speed.</p>
+            <h3 class="p-matrix__title"><a href="https://snapcraft.io/spotify">Spotify</a></h3>
+            <p class="p-matrix__desc">Stream the tracks you love instantly, browse the chartsor fire up ready made playlists in every genre and mood.</p>
           </div>
         </li>
         <li class="p-matrix__item">
-          <img class="p-matrix__img" src="{{ ASSET_SERVER_URL }}171fcf42-Chromium_11_Logo.svg?fmt=png&w=60" alt="Chromium icon" />
+          <a href="https://snapcraft.io/skype">
+            <img class="p-matrix__img" src="https://dashboard.snapcraft.io/site_media/appmedia/2017/11/Skype.png" alt="Skype icon" />
+          </a>
           <div class="p-matrix__content">
-            <h3 class="p-matrix__title">Chromium</h3>
-            <p class="p-matrix__desc">A fast, simple and secure web browser, built for the modern web.</p>
+            <h3 class="p-matrix__title"><a href="https://snapcraft.io/skype">Skype</a></h3>
+            <p class="p-matrix__desc">Skype keeps the world talking. Say &ldquo;hello&rdquo; with an instant message, voice or video call &mdash; all for free.</p>
           </div>
         </li>
-        <li class="p-matrix__item">
-          <img class="p-matrix__img" src="{{ ASSET_SERVER_URL }}6b138bd2-thunderbird-icon.png" alt="Thunderbird icon" />
+        <li class="p-matrix__item last-row">
+          <a href="https://snapcraft.io/vlc">
+            <img class="p-matrix__img" src="https://dashboard.snapcraft.io/site_media/appmedia/2016/07/vlc.png" alt="VLC icon" />
+          </a>
           <div class="p-matrix__content">
-            <h3 class="p-matrix__title">Thunderbird</h3>
-            <p class="p-matrix__desc">Terrific email application, from Mozilla, that&rsquo;s easy to set up and customise.</p>
+            <h3 class="p-matrix__title"><a href="https://snapcraft.io/vlc">VLC player</a></h3>
+            <p class="p-matrix__desc">No other video player is compatible with as many different file formats.</p>
           </div>
         </li>
         <li class="p-matrix__item">
-          <img class="p-matrix__img" src="{{ ASSET_SERVER_URL }}34214609-dropbox-icon.png" alt="Dropbox icon" />
+          <a href="https://snapcraft.io/firefox">
+            <img class="p-matrix__img" src="https://dashboard.snapcraft.io/site_media/appmedia/2018/02/firefox256.png" alt="Firefox icon" />
+          </a>
           <div class="p-matrix__content">
-            <h3 class="p-matrix__title">Dropbox</h3>
-            <p class="p-matrix__desc">The world&rsquo;s favourite cloud backup and file sharing service.</p>
+            <h3 class="p-matrix__title"><a href="https://snapcraft.io/firefox">Firefox</a></h3>
+            <p class="p-matrix__desc">The speedy, independent, open source browser from Mozilla.</p>
```

Comment:
Should be `Firefox Quantum. Fast for good. Now 2x faster and 30% lighter than Chrome.`